### ISSUE TITLE
project page has no prominent link back to home/search

### ISSUE THREAD

**Issue by n8han** [softprops/ls#16]
http://ls.implicit.ly/unfiltered/unfiltered

I generally expect a top left link to go back to the home page, but here it says "unfiltered" and indeed it just goes to the page I'm already on.

An easy solution would be to have it read "ls> unfiltered"

and at least the first three characters could link back to the home page. (They could also be magenta/salmon/whatever-that-is!) I'm not sure if any link to "unfiltered" is helpful since we are already there, but I could be missing something.

Related, If there were a search box at the top of every page I would not care about going home so much, although I still think it's probably a good idea to have a link.

**Comment by softprops** [comment]
Working on this tonight

**Comment by softprops** [comment]
should be fixed and pushed to heroku now

**Comment by n8han** [comment]
Yeah, looks awesome. Clear to launch, if you ask me!

**Comment by softprops** [comment]
I'm doing some more local testing first. I just added conflict detection in the plugin. Tonight is looking like the right time to announce. I've got no read be but Im just going to reference the info on the site. Im also thinking of making a followup pamflet gh page, possibly tying together pamflet and josh's gh-pages plugin

**Comment by n8han** [comment]
What repo are you going to publish to? I can give you the password for databinder.net at Scala-tools if you want.
Nathan

doug tangren reply@reply.github.com wrote:

I'm doing some more local testing first. I just added conflict detection in the plugin. Tonight is looking like the right time to announce. I've got no read be but Im just going to reference the info on the site. Im also thinking of making a followup pamflet gh page, possibly tying together pamflet and josh's gh-pages plugin

---
